InstabugSample/src/screens/HomeScreen.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { StyleSheet, Text } from 'react-native';
+import Instabug, {
+  BugReporting,
+  CrashReporting,
+  FeatureRequests,
+  Replies,
+  Surveys,
+} from 'instabug-reactnative';
+
+import Button from '../components/Button';
+import Screen from '../components/Screen';
+
+function HomeScreen() {
+  const invoke = () => Instabug.show();
+
+  const showMultipleQuestionSurvey = () => Surveys.showSurvey('95s5cjU1i74m23h0M9t_Sg');
+
+  const showNpsSurvey = () => Surveys.showSurvey('CHUJHGRx8s1qPcCSs85kFA');
+
+  const showFeatureRequests = () => FeatureRequests.show();
+
+  const sendBugReport = () => BugReporting.show(BugReporting.reportType.bug);
+
+  const sendCrashReport = () => {
+    try {
+      throw new Error('Handled Exception From Instabug Test App');
+    } catch (Exception) {
```

Comment:
**suggestion:** PascalCase is confusing here

```suggestion
    } catch (exception) {
```

Reply:
I agree, I think we should also rename it to `error` or `err` as a commonly used convention by JS devs.

Reply:
Let's go with `err`? it feels more javascript-y